core/templates/css/oppia.css

```diff
@@ -533,6 +533,10 @@ p:last-child {
    CSS class, but we do not know what JS code is causing it to be inlined in the
    span tag. There is as such no way to remove !important as of now, it's a
    CKEditor bug (issue #6206 on Oppia). */
+.oppia-welcome-guide-browser-image span {
```

Comment:
Add this after 542, see the comment above on line 533. It's meant for the next css block.